### ISSUE TITLE
Bad interactions with the system player started from a web view

### ISSUE THREAD

**Issue by defagos** [SRGSSR/srgletterbox-apple#194]
# Playback issues

According to discussions with other teams, it appears that starting playback in a web view (with the system player) changes the playback state of a Letterbox controller to pause, then play, after which the controller remains in a somehow inconsistent state.

Context:

- iOS 13
- Controller registered with the Letterbox service
- Audio session set to playback

# Control center issues

For applications in which videos can be played from a web view, subsequent playbacks with Letterbox will not display control center information anymore.

This behavior is incorrect, but might be related to an iOS regression. Doing the same with iOS 10 namely works.

To verify whether this is an iOS issue, we should use a simple project with both an `AVPlayer` and control center integration, as well as a web view.

### Issue type

Incorrect behavior

### Environment information

* Library version: Tested with 1.15
* iOS version: 13
* Letterbox 1.5
* Device: Any, but physical

### Reproducibility

Always reproducible

### Steps to reproduce

With an app mixing Letterbox native playback and webview playback:

1. Play some media with Letterbox, check integration with the control center.
1. Play a video from the web view, in full screen.
1. Play another media with Letterbox. No control center integration is available anymore.

**Comment by defagos** [comment]
We should first create a small example in SRG Media Player demo directly. The issue is probably there.

**Comment by defagos** [comment]
Starting with the Terms & conditions page, and using the burger menu in Play RTS, it is quite easy to get to a page containing a video, e.g. _Travailler à la RTS_. I used the following test scenario:

- Open a media with the native Letterbox player. Check that control center integration is correct.
- Open a video from a web page. Check control center integration (basic Apple integration).
- Open the same media as before with the native Letterbox player again. Control center integration should be correct.

I repeated this test with the following versions:

- Play 2.9.7-322 (Letterbox 2.0.0)
- Play 2.9.7-325 (Letterbox 3.0.0)

The bug is clearly visible with 322, fixed with 325. It is also fixed in the current Play 3.0.0 (Letterbox 4.0.0). I conclude that the control center refactoring of 3.0.0 fixed this issue.
